reference/api/pubsub_api.md

```diff
@@ -61,6 +61,7 @@ Example:
 ```json
 [
   {
+    "pubsubname": "pubsub",
```

Comment:
It's `pubsubName`.

Reply:
Took it from our current [docs](https://github.com/dapr/docs/blob/master/howto/consume-topic/README.md#subscribe-to-topics), so, sounds like we may need a global review